cl_dll/in_camera.cpp

```diff
@@ -15,8 +15,10 @@
 #include "camera.h"
 #include "in_defs.h"
 #include "Exports.h"
+#include "pm_shared.h"
 
 #include "port.h"
+#include "demo_api.h"
```

Comment:
both headers needed?

Reply:
yep, one for obs server mode macros, one for pDemoAPI struct